### ISSUE TITLE
[Bug]: Complete package information is not visible at different zooming conditions (125%, 150%, 175%, 200%).

### ISSUE THREAD

**Issue by donnie-msft** [NuGet/Home#11119]
### NuGet Product Used

Visual Studio Package Management UI

### Product Version

VS any

### Worked before?

no

### Impact

It bothers me. A fix would be nice

### Repro Steps & Context

See https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1359969

OS & Environment Details:
Microsoft Windows 10 Enterprise Version 21H1 (OS build: 19043.1052)
Microsoft Visual Studio Enterprise 2022 Preview (64-bit)
Version 17.0.0 Preview 1.0 [31404.170.main]
Repro Steps:​
Change the screen display resolution to an "average" mid-range resolution (1280*768 or near to it 1280x800)
Set the slider to any zooming condition (125%, 150%, 175%, 200%) under 'Make text bigger' in Display settings in Ease of Access.
Launch latest Visual studio 2022 preview 1
Tab till 'Create new prokect' and press enter
Search for "gRPC Service" project and select it 
Tab till 'Next' button and press enter
Provide proper details and press enter on 'Create' button
Tab till "Solution" and open context menu from keyboard
Tab till 'Manage Nuget Packages for solution' and press enter
Tab till 'Installed/Browse' tab item
Tab till any package having larger text
Verify whether complete package information is visible at different zooming conditions or not.
Actual Result:​
Complete package information is not visible at different zooming conditions (125%, 150%, 175%, 200%).
Expected Result:​
Complete package information must be visible at different zooming conditions (125%, 150%, 175%, 200%).
User Impact: ​
Low vision users may not able to know complete package information if it is not visible at different zooming conditions.
MAS Reference:​
MAS 1.4.4 Resize Text
